plugin/storage/grpc/config/config.go

```diff
@@ -106,12 +110,13 @@ func (c *Configuration) buildRemote(logger *zap.Logger, tracerProvider trace.Tra
 		opts = append(opts, grpc.WithUnaryInterceptor(tenancy.NewClientUnaryInterceptor(tenancyMgr)))
 		opts = append(opts, grpc.WithStreamInterceptor(tenancy.NewClientStreamInterceptor(tenancyMgr)))
 	}
-	conn, err := grpc.DialContext(ctx, c.RemoteServerAddr, opts...)
+	var err error
+	c.remoteRPCClient, err = grpc.DialContext(ctx, c.RemoteServerAddr, opts...)
```

Comment:
this is not a client
```suggestion
	c.remoteConn, err = grpc.DialContext(ctx, c.RemoteServerAddr, opts...)
```